Formula/jdtls.rb

```diff
@@ -16,8 +16,8 @@ class Jdtls < Formula
     sha256 cellar: :any_skip_relocation, all: "5af62f9cf5c42b7ba320b51901224ca87593475a80d292beeb3cbd9aede5e50f"
   end
 
-  depends_on "openjdk@11"
-  depends_on "python@3.9"
+  depends_on "openjdk"
+  depends_on "python@3.10"
```

Comment:
`libexec/"bin/jdtls"` uses `#!/usr/bin/env python3`, so it won't work as is with `python@3.10`. This should end up falling back to Python 3.8 on macOS, but script needs Python 3.9 at minimum.

Can either do a shebang rewrite, or a PATH override in env script.

Not sure if this is what is causing current test failure though.